ui/packages/catalog-ui-search/src/main/webapp/component/query-time/query-time.view.js

```diff
@@ -120,6 +107,29 @@ module.exports = Marionette.LayoutView.extend({
         this.$el.toggleClass('is-timeRange-between', timeRange === 'between');
         this.$el.toggleClass('is-timeRange-relative', timeRange === 'relative');
     },
+    setupTemporalSelections: function () {
+        var metacards = metacardDefinitions.sortedMetacardTypes.filter((metacard) => metacard.type === "DATE");
```

Comment:
❗️ We'll need to take into account the admin's configuration for hidden attributes as well as their aliases.

Reply:
Would that be using the `filter-builder` instead?